src-docs/src/views/package/changelog.js

```diff
@@ -4,7 +4,7 @@ import { EuiMarkdownFormat } from '../../../../src';
 import { GuidePage } from '../../components/guide_page';
 
 const changelogSource = require('!!raw-loader!../../../../CHANGELOG.md').default.replace(
-  /## \[`main`\].+?##/s, // remove the `main` heading & contents
+  /## \[`main`\].+?##(?= \[`\d)/s, // remove the `master` heading & contents
```

Comment:
Looks like a bad merge conflict fix with the `main` rename efforts.

```suggestion
  /## \[`main`\].+?##(?= \[`\d)/s, // remove the `main` heading & contents
```

Looking at the changelog entry, I'm not actually sure these regexes changes are needed. We're using `###` for the new Feature announcement and not `##`. @thompsongl am I missing something here?

Reply:
I did try the simpler lookahead, but it didn't work: https://regexr.com/69qu7

Reply:
Ahhh super weird! That regex feels like it's treating the empty space like a `.`, . Any thoughts on:

```## \[`main`\].+?##(?= \W)```

Or does this feel less specific/more confusing than what's already there? No worries if so, feel free to disregard